### ISSUE TITLE
Something weird is happening, Help please

### ISSUE THREAD

**Issue by UKPMAN** [FlailingFog/KK-Blender-Porter-Pack#579]
I have Koikatsu Sunshine Better Repack R3.1 and I installed KKPB_Exporter, when I go to the character maker and try exporting the character pose just goes to a H-pose and not the T-pose.

**Like This:-**
![Screenshot 2024-07-17 185113](https://github.com/user-attachments/assets/e3822076-3143-4fec-9d17-efc4ad93e5f1)

but it is rigged in Blender, but the rig controls are messed up. There is no T-pose in my game either. If I use the "Keep current Pose" it exports in that pose but there's a absurd amount of bones and doesn't rig.
![rsdfgh](https://github.com/user-attachments/assets/d9d9f55b-1603-4232-b847-3d1d08b7bd95)

In Blender:-
![Screenshot 2024-07-17 200949](https://github.com/user-attachments/assets/818ace08-5c3e-4e63-9633-77e5bd59f6c6)

![image](https://github.com/user-attachments/assets/a421a89f-011f-445b-98d7-b4ccac4c7eef)

**Comment by FlailingFog** [comment]
You have to install the KKS HF patch, or use the guidance provided here: https://github.com/FlailingFog/KK-Blender-Porter-Pack/issues/523

**Comment by UKPMAN** [comment]
@FlailingFog The problem was that my game didn't have the T-pose on my base game, so I installed a mod that adds the T-pose and it works now. I also got the betterRepack R10 now. 

But I have to use the porterPack 6.5.0, if I use the 6.6.3 I get a error about "ShadowColor". 
With the older addon and exporter this works fine but sometimes some materials on hair or accessories are just white.
![Screenshot 2024-07-31 010419](https://github.com/user-attachments/assets/31bde92c-b2ea-4e9d-a7e6-8750ec23c003)

Even in the older exporter I have to untick "Export Hit meshes" or I'll get this when importing to blender:
![image](https://github.com/user-attachments/assets/7d5d406c-954e-4daa-b928-bd57d7408249)


This is what happens if I use the newest(6.6.3) version of PorterPack:
![image](https://github.com/user-attachments/assets/50373139-81c7-47b5-bcac-0f1e1eca6f98)

Is there any "FIX" or something I can do?
